src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -85,6 +85,7 @@ class ZHAConfigDashboard extends LitElement {
         <a href="/config/zha/add" slot="fab">
           <mwc-fab
             title=${this.hass.localize("ui.panel.config.zha.add_device")}
+            label=${this.hass.localize("ui.panel.config.zha.add_device")}
```

Comment:
We should not have a title and a label

Reply:
But then we either miss the tooltip or the internal aria-label support for screen readers:
- `title` is used to set the tooltip
- `label` is used by mwc for aria labels

To limit the number of strings we pass around we would need to create something like `HaFab` where we only pass one property and then use it internally to set both `title` and `label`.

Reply:
If however we combine my changes with the ones from https://github.com/home-assistant/frontend/pull/7612 we would not really need a tooltip anymore if we always use the `extended` mode = show a text in the frontend.